For testing. Runs the test from a different environment than the Mozilla
default.

diff --git a/python/mozbuild/mozbuild/test/test_base.py b/python/mozbuild/mozbuild/test/test_base.py
--- a/python/mozbuild/mozbuild/test/test_base.py
+++ b/python/mozbuild/mozbuild/test/test_base.py
@@ -393,5 +393,5 @@
         self.assertEqual(p.srcdir_path(), "/src/foo/bar")
         self.assertEqual(p.objdir_path(), "/src/obj/foo/bar")
 
-if __name__ == '__main__':
-    main()
+#if __name__ == '__main__':
+#    main()
diff --git a/python/mozbuild/mozbuild/test/test_mozconfig.py b/python/mozbuild/mozbuild/test/test_mozconfig.py
--- a/python/mozbuild/mozbuild/test/test_mozconfig.py
+++ b/python/mozbuild/mozbuild/test/test_mozconfig.py
@@ -477,5 +477,5 @@
             self.assertEquals(e.exception.output, ['hello world'])
 
 
-if __name__ == '__main__':
-    main()
+#if __name__ == '__main__':
+#    main()
